packages/apollo-server-azure-functions/README.md

```diff
@@ -138,6 +138,26 @@ The options correspond to the [express cors configuration](https://github.com/ex
 * `credentials`: boolean
 * `maxAge`: number
 
+### Request Body / Token
+By default Azure Functions will pass the request object as req.  Pass this into the context of the ApolloServer constructor to access request contents.  For example for passing an auth token to a backend.
+
+```js
+const server = new ApolloServer(
+    { typeDefs,       
+        resolvers,
+        context: (req) => {
+          const authHeader = req.context.bindings.req.headers.authorization || '';
```

Comment:
Is that actually the best way to get a header? I see at https://www.apollographql.com/docs/apollo-server/api/apollo-server/#middleware-specific-context-fields that we pass `context` and also `request`. Wouldn't `req.request.headers` be simpler?